### PR TITLE
V225-017: Display gnatpp messages if any

### PR DESCRIPTION
**Pull request by AnthonyLeonardoGracio**

### CODE REVIEW

File: source/ada/lsp-ada_handlers.ads [hunk 1]
```diff
@@ -67,11 +67,13 @@ package LSP.Ada_Handlers is
      (Self : Message_Handler'Class;
       File : Virtual_File) return LSP.Messages.DocumentUri;
    --  Turn Virtual_File to URI
+   --  TODO: put this somewhere else without Self : Message_Handler
```
Comment (AdrienBoulanger) [inline source/ada/lsp-ada_handlers.ads:70]: TODO?

File: source/ada/lsp-ada_contexts.adb [hunk 2]
```diff
@@ -761,7 +762,8 @@ package body LSP.Ada_Contexts is
         (Context => Self,
          Span    => Span,
          Cmd     => Self.PP_Options,
-         Edit    => Edit);
+         Edit    => Edit,
```
Comment (AdrienBoulanger) [inline source/ada/lsp-ada_contexts.adb:765]: Align issue

**Review by AdrienBoulanger — CHANGES_REQUESTED** [review]

File: source/ada/lsp-ada_handlers.ads (hunk 1 above)
Comment (AnthonyLeonardoGracio) [reply source/ada/lsp-ada_handlers.ads]: Done

File: source/ada/lsp-ada_contexts.adb (hunk 2 above)
Comment (AnthonyLeonardoGracio) [reply source/ada/lsp-ada_contexts.adb]: Done

**Review by AdrienBoulanger — APPROVED** [review]

**Pull request merged**